### PR TITLE
Umapa crawler.

### PR DESCRIPTION
**Pull request by moorchegue**
Addresses #63

### CODE REVIEW

File: scraping/spiders/umapa.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    url = "https://www.mapotic.com/api/v1/maps/10392/public-pois/{}/"
+    return url.format(point["properties"]["id"])
+
+
+def build_point(response):
+    point = response.json()
+    lat, lng = point["point"]["coordinates"]
+    cat = CATEGORIES.get(point["category"]["name"]["en"], DEFAULT_CATEGORY)
+    return {
+        "country": "",
```
Comment (littlepea) [inline scraping/spiders/umapa.py:71]: Isn't country always Poland for this one?

**Review by littlepea — APPROVED** [review]

File: scraping/spiders/umapa.py (hunk 1 above)
Comment (moorchegue) [reply scraping/spiders/umapa.py:71]: The largest amount of points is Czech, but there's several more countries, all have significant number of points.

Comment (littlepea) [reply scraping/spiders/umapa.py:71]: I see, ok

Comment (moorchegue) [reply scraping/spiders/umapa.py:71]: There's a lot of meta info, but country and city are never to be found. Perhaps with the growing amount of incomplete geoinfo points we could do some offline geocoding?

E.g. https://github.com/richardpenman/reverse_geocode

Comment (littlepea) [reply scraping/spiders/umapa.py:71]: Yes, I'll handle the missing data and reverse geocoding in post-processing with `main.py`

**Pull request merged**